src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_NAME_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] dataFromFile = readFromFile(fromFileName).split(" ");
+        String report = createReport(dataFromFile);
+        writeToFile(toFileName, report);
+    }
+
+    public String createReport(String[] dataFromFile) {
```

Comment:
Please make all methods except getStatistic() private as they are not intended for external usage